rust-webvr/src/api/googlevr/service.rs

```diff
@@ -17,8 +16,7 @@ const SERVICE_CLASS_NAME:&'static str = "com/rust/webvr/GVRService";
 pub struct GoogleVRService {
     ctx: *mut gvr::gvr_context,
     controller_ctx: *mut gvr::gvr_controller_context,
-    displays: Vec<GoogleVRDisplayPtr>,
-    gamepads: Vec<GoogleVRGamepadPtr>,
+    display: Option<GoogleVRDisplayPtr>,
```

Comment:
This seems like an odd change, does Google VR really guarantee only one display?

Reply:
From the API it seems so?

Either way, our API usage guarantees one display, so the question is moot for the purposes of this PR.